### ISSUE TITLE
Boolean Disjoint not detecting intersection

### ISSUE THREAD

**Issue by Necr0** [Turfjs/turf#1337]
### What I did:
I tried calling the `turf.booleanDisjoint` on two rectangular polygons that quite clearly intersect.
https://gist.github.com/Necr0/11ac9e7bc0c84a20a85b2a7c948f38bf#file-turf-testdata-geojson (geojson)
https://jsfiddle.net/p7sjaozk/3/ (turf.booleanDisjoint test)

### What I expected:
The method should return false.

### What happened:
It returned true.

**Comment by rowanwins** [comment]
Hi @Necr0 

This has been resolved in v6 of booleanDisjoint which is currently available as a standalone module, however the entirely bundled version of turf is still running as v5 and hasn't incorporated the relevant change.

To use v6 
````
npm install @turf/boolean-disjoint

// and then
var booleanDisjoint = require('@turf/boolean-disjoint').default
````

Was originally raised in #1216 and fixed by #1222 

If you're happy with that feel free to close the issue

**Comment by Necr0** [comment]
Okay, sorry for reporting it then. I used browserify to bundle a version containing the new version myself.
